resources/views/partials/servicerecord.blade.php

```diff
@@ -22,8 +22,8 @@
                         Assigned: {{$user->getPrimaryDateAssigned()}}
                     </div>
                     @if(($permsObj->hasPermissions(['EDIT_SELF']) && Auth::user()->id == $user->id) || $permsObj->hasPermissions(['EDIT_MEMBER']) || $permsObj->isInChainOfCommand($user))
-                        <div class="Incised901Light seventy-five">
-                            Path: {{Form::select('path', ['service' => 'Service', 'staff' => 'Staff', 'line' => 'Line'], $user->path, ['id' => 'path', 'data-id' => $user->id])}}
+                        <div class="Incised901Light">
+                            Path: {{Form::select('path', ['service' => 'Service', 'staff' => 'Staff', 'line' => 'Line'], $user->path, ['id' => 'path', 'data-id' => $user->id, 'title' => 'Click to change path'])}} <span class="fa fa-info-circle info" title="Click your path to change it"></span>
```

Comment:
Make it bigger, add a tool tip to the select input and add the info icon with it's own tool tip.  I hate end users :-/